src/components/StatusTradeRequests.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { gql, graphql, compose } from 'react-apollo';
+import ToggleSwitch from '@trendmicro/react-toggle-switch';
+
+const StatusTradeRequests = (props) => {
```

Comment:
Let's rename this file/component to "ResolvedTradeRequests" since the other component is named "PendingTradeRequests".